src/black/mode.py

```diff
@@ -177,6 +177,7 @@ class Preview(Enum):
     wrap_long_dict_values_in_parens = auto()
     multiline_string_handling = auto()
     typed_params_trailing_comma = auto()
+    docstring_check_for_newline = auto()
```

Comment:
Could you document this in future-style.md? (I have a PR in #4187 that will enforce this.)

Reply:
Yep, it is in there now. I do apologize for missing that.